tests/search/components/TestSearchForm.js

```diff
@@ -7,24 +7,30 @@ const render = ReactTestUtils.renderIntoDocument;
 const findByTag = ReactTestUtils.findRenderedDOMComponentWithTag;
 
 describe('<SearchForm />', () => {
+  var onSearch;
+  var root;
+  var form;
+  var input;
+
+  beforeEach(() => {
+    onSearch = sinon.spy();
+    root = render(<SearchForm onSearch={onSearch} />);
+    form = findByTag(root, 'form');
```

Comment:
`form` and `input` could be pulled from `root.refs.form` and `root.refs.input`, too. That's how `query` is found in `it calls onSearch with a search query`.